src/pages/youthboard/surveys/index.tsx

```
@@ -29,6 +32,21 @@ const Survey = () => {
     router.push('/youthboard/volunteerList');
   };
 
+  const formFields = [
```

ESLint

[error] 'formFields' is assigned a value but never used.

(no-unused-vars)